library/smarty/plugins/function.amcCollect.php

```diff
@@ -1,42 +0,0 @@
-<?php
```

Comment:
These have absolutely nothing to do with the PR. 
It got lumped in and I don't know why.  
I'll do a new pull request